### PR TITLE
Add an IsFolder method to FileOrFolderInformation class.

### PR DESCRIPTION
**Pull request by WilliamsJason**

### CODE REVIEW

**Comment by WilliamsJason** [comment]
Fixes issue #205.

File: WindowsDevicePortalWrapper/WindowsDevicePortalWrapper.Shared/Core/AppFileExplorer.cs [hunk 1]
```diff
@@ -311,6 +311,11 @@ public class FileOrFolderInformation
             public long SizeInBytes { get; private set; }
 
             /// <summary>
+            /// Gets whether the current item is a folder
+            /// </summary>
+            public bool IsFolder { get { return this.Type == 0x10; } }
```
Comment (hpsin) [inline WindowsDevicePortalWrapper/WindowsDevicePortalWrapper.Shared/Core/AppFileExplorer.cs]: Should be 
  return ((this.Type & 0x10) == 0x10) // check for 0x10 flag
As the type is a set of flags, defined here, where 0x10 is the flag for directory.  Other flags get added on that can change it (e.g. MyMusic is 9238)

Comment (hpsin) [inline WindowsDevicePortalWrapper/WindowsDevicePortalWrapper.Shared/Core/AppFileExplorer.cs]: https://msdn.microsoft.com/en-us/library/windows/desktop/gg258117(v=vs.85).aspx

Comment (WilliamsJason) [inline WindowsDevicePortalWrapper/WindowsDevicePortalWrapper.Shared/Core/AppFileExplorer.cs]: Ah of course, thank you!

**Comment by hpsin** [comment]
Thanks!

**Pull request merged**